warehouse/query-core/src/main/java/datawave/query/jexl/visitors/ExpandMultiNormalizedTerms.java

```diff
@@ -317,10 +318,15 @@ protected JexlNode expandNodeForNormalizers(JexlNode node, Object data) {
                     Set<String> normalizedTerms = Sets.newHashSet();
                     List<JexlNode> normalizedNodes = Lists.newArrayList();
                     boolean failedNormalization = false;
+                    boolean regexNode = (node instanceof ASTNRNode || node instanceof ASTERNode);
                     // Build up a set of normalized terms using each normalizer
                     for (Type<?> normalizer : dataTypes) {
                         try {
                             if (normalizer instanceof OneToManyNormalizerType && ((OneToManyNormalizerType<?>) normalizer).expandAtQueryTime()) {
+                                if (regexNode) {
+                                    throw new IllegalArgumentException(
+                                                    "OneToManyNormalizers to not handle regex normalization: " + fieldName + " -> " + normalizer.getClass());
```

Comment:
this is no longer accurate.  there are some one-to-many normalizers which can and probably should now.